Sources/DNA/Color/Color.swift

```diff
@@ -65,10 +65,42 @@ import UIKit
         return UIColor(r: 246, g: 248, b: 251)!
     }
 
+    public class var midnightBackground: UIColor {
+        return UIColor(hex: "1D1D26")
+    }
+
+    public class var midnightSectionHeader: UIColor {
+        return UIColor(hex: "585E8A")
+    }
+
+    public class var midnightSectionSeparator: UIColor {
+        return UIColor(hex: "34343E")
+    }
+
     // swiftlint:disable:next identifier_name
     convenience init?(r: CGFloat, g: CGFloat, b: CGFloat, a: CGFloat = 1.0) {
         self.init(red: r / 255.0, green: g / 255.0, blue: b / 255.0, alpha: a)
     }
+
+    /// Base initializer, it creates an instance of `UIColor` using an HEX string.
+    ///
+    /// - Parameter hex: The base HEX string to create the color.
+    convenience init(hex: String) {
```

Comment:
Why use `String` when we can use `Int`?

Reply:
Hex strings are in the form of "1D1D26" or "#1D1D26" so Int wouldn't be a good fit.